src/core/plugins/swagger-js/index.js

```diff
@@ -30,7 +30,9 @@ export default function({ configs, getConfigs }) {
     },
     statePlugins: {
       configs: {
-        wrapActions: configsWrapActions
+        wrapActions: {
```

Comment:
This is technically an invalid usage of `wrapActions` interface. The code here creates unsupported nested wrapActions object:

```js
wrapAction: {
  configsWrapActions: {
    loaded: ...implementation...
  }
}